### PR TITLE
Priority tag issue resolved

### LINKED ISSUE #16: XML Document structure - priority
There is issue with tags in the xml., when checking priority changefreq tag is set

Here is what needs to change

https://github.com/amannn/multi-sitemap/pull/17

Could you publish new version to npm?

Thanks in advance

### PR DESCRIPTION
**Pull request by mpochwala**
Closes #16

### CODE REVIEW

File: src/SitemapWriterXml.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default class SitemapWriterXml extends SitemapWriterFile {
                 `<lastmod>${formatLastModified(entry.lastModified)}</lastmod>`,
               entry.changeFrequency &&
                 `<changefreq>${entry.changeFrequency}</changefreq>`,
-              entry.priority && `<changefreq>${entry.priority}</changefreq>`,
+              entry.priority && `<priority>${entry.priority}</priority>`,
```
Comment (amannn) [inline src/SitemapWriterXml.ts:50]: Great point, thanks for noticing and fixing this! Can you add a test for this?

Comment (mpochwala) [reply src/SitemapWriterXml.ts:50]: done

**Review by amannn — APPROVED** [review]
Thanks a lot!

**Comment by amannn** [comment]
Published in 0.4.1

**Pull request merged**